package dio.me.desafiofinal.santander.application.services.exceptions.dto;

public record GivenDateInvalidExceptionDTO(String model, String message) {
}
